### ISSUE TITLE
Error Cannot read property 'selectedControl' of undefined

### ISSUE THREAD

**Issue by cristianurbano** [MacKentoch/easyFormGenerator#63]
Hi!

We have two errors with the **2.1.1 version**.

1- `Formly Error: There is no type by the name of "none"`  -> When trying to add a new column.
2- `Cannot read property 'selectedControl' of undefined` -> When trying to open the control modal for this new column.

----
### Steps to reproduce this error:

1. Checkout to v2.1.1
2. npm install && npm start
3. Go to `http://localhost:8080/preview/stepway.html`
4. Add a column in some row. **(you get the first error here)**
5. Click on the new column to add a control **(you get the second error here)**

### Screenshots
![error1](https://cloud.githubusercontent.com/assets/2442937/21851852/9e24766c-d7ef-11e6-93bc-06bce1d9d70c.png)
![error2](https://cloud.githubusercontent.com/assets/2442937/21851857/a11a9f68-d7ef-11e6-95de-09906826a652.png)

**Comment by vinnybod** [comment]
I am also getting this error, however it seems to work fine in the stepway demo provided with the npm module, so I am trying to debug now

**Comment by MacKentoch** [comment]
Could try with minified version? 

Demo uses min version.

**Comment by vinnybod** [comment]
using minified
```
TypeError: Cannot read property 'selectedControl' of undefined
    at e.value
```

**Comment by MacKentoch** [comment]
I don't understand...

It should bug here but it doesn't (even no minified version).
I can't reproduce this bug.

I hate angular errors for not helping...

I guess it is a matter of dependencies?

**Comment by vinnybod** [comment]
I'll experiment a bit with my dependencies and see if i can fix it or get a different error.

**Comment by MacKentoch** [comment]
Thanks for help.

Searching this bug (still not found) **I fixed another bug** with `basic select` while editing in modal.
This bug prevented from manipulating options (in the table: change order, or delete...).

**Comment by cristianurbano** [comment]
@MacKentoch  I added the steps to reproducing this error

**Comment by vinnybod** [comment]
@MacKentoch I can confirm that I got the same issue in the demo following these steps.

**Comment by MacKentoch** [comment]
v2.1.2 fixes #64 but this #63 I don't reproduce it.

What angular js version do you use for example?

**Comment by cristianurbano** [comment]
@MacKentoch I used this repo with the version of the package.json

**Comment by vinnybod** [comment]
@MacKentoch have you tried to reproduce with the new steps provided? Perhaps when you made your fixes for v2.1.2, it was fixed?
